crates/forge/bin/cmd/test/mod.rs

```diff
@@ -115,6 +118,10 @@ pub struct TestArgs {
     #[arg(long, help_heading = "Display options")]
     json: bool,
 
+    /// Output test results as JUnit XML report.
+    #[arg(long, conflicts_with = "json", help_heading = "Display options")]
+    junit: bool,
```

Comment:
I think we should have a global output-format flag enum

Reply:
that makes sense

Reply:
would this be a breaking change for users of `--json` flag or we should not bother about?

Reply:
hide it and treat the same as format=json

Reply:
did that and it looks nice, but it conflicts with snapshot hidden format arg
https://github.com/foundry-rs/foundry/blob/fb9dbba4ed81321557a68053f845ca6ea46b6ba1/crates/forge/bin/cmd/snapshot.rs#L52-L55

looks like is not even used as there's only one option, should it just be removed?

Reply:
Yeah it's been hidden for a while https://github.com/foundry-rs/foundry/commit/3f13a986e69c18ea19ce634fea00f4df6b3666b0